src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationNotifications.cs

```diff
@@ -23,7 +23,6 @@ public OpenIdConnectAuthenticationNotifications()
             MessageReceived = notification => Task.FromResult(0);
             SecurityTokenReceived = notification => Task.FromResult(0);
             SecurityTokenValidated = notification => Task.FromResult(0);
-            RedirectToIdentityProvider = notification => Task.FromResult(0);
```

Comment:
Once we decide on the pattern to use everywhere, one single PR should apply the changes everywhere.